bundles/org.openhab.binding.speedtest/src/main/java/org/openhab/binding/speedtest/internal/SpeedtestHandlerFactory.java

```diff
@@ -37,6 +41,13 @@
 public class SpeedtestHandlerFactory extends BaseThingHandlerFactory {
 
     private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Set.of(THING_TYPE_SPEEDTEST);
+    private final TimeZoneProvider timeZoneProvider;
+
+    @Activate
+    public SpeedtestHandlerFactory(@Reference HttpClientFactory httpClientFactory,
```

Comment:
Please remove reference to HttpClientFactory  as finally you do not use it.

Reply:
did it